jest.setup.js

```diff
@@ -0,0 +1,4 @@
+const {TextEncoder, TextDecoder} = require('util');
+
+global.TextEncoder = TextEncoder;
+global.TextDecoder = TextDecoder;
```

Comment:
These are already globals in Node
![image](https://github.com/croct-tech/sdk-js/assets/11063910/758a66d5-1600-462f-a3c7-844090c17fa5)

Reply:
For some reason, it's not available on the Jest environment.